docs/community-operator-pipeline.md

```diff
@@ -0,0 +1,40 @@
+# community-operator-pipeline
+
+The community operator pipeline provides a platform for openshift community contributors to validate, share, and distribute openshift operators.
+
+**community-hosted-pipeline**
+
+The community hosted pipeline is used to certify the Operator bundles.
+
+It’s an additional layer of validation that has to run within the Red Hat infrastructure. It is intended to be triggered upon the creation of a bundle pull request and successfully completes with merging it. As part of the community hosted pipeline, the operator submitted in the PR will be tested via preflight trigger. 
+
+The community hosted pipeline run results will be posted in the github PR comment.
+
+![community-hosted-pipeline](images/op_test_pr.png)
+
+**community-release-pipeline**
+
+The community release pipeline is responsible for releasing a bundle image which has passed certification.
+It's intended to be triggered by the merge of a bundle pull request by the hosted pipeline.
+It successfully completes once the bundle has been distributed to all relevant Operator catalogs
+and appears in the Red Hat Ecosystem Catalog.
```

Comment:
The bundle doesn't appear on the Ecosyste Catalog. Ecosystem Catalog is a frontend application that only displays Red Hat and ISV content.